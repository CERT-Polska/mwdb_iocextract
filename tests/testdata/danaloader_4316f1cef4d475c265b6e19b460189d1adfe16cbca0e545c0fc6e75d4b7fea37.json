{
    "type": "danaloader",
    "urls": [
        {
            "url": "http://92.63.197.60/crabin.exe?NSIjJOd"
        }
    ]
}