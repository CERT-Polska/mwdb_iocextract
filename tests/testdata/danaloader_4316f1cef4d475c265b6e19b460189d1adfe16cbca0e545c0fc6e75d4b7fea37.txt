NetLoc http://92.63.197.60/crabin.exe?NSIjJOd
