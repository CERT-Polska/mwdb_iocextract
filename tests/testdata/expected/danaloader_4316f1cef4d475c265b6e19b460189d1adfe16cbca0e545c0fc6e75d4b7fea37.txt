NetLoc 92.63.197.60//crabin.exe
